src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,7 @@
 package core.basesyntax;
 
 public class MainApp {
+String[] Machine = {"Truck", "Bulldozer", "Excavator"};
```

Comment:
The variable name 'Machine' should start with a lowercase letter to follow Java naming conventions. Also, it seems like this array is meant to store instances of different machines rather than strings. Consider creating an array of type 'Machine' and initializing it with instances of 'Truck', 'Bulldozer', and 'Excavator'.